### PR TITLE
jcliff.spec

### PR DESCRIPTION
**Pull request by hcherukuri**

### CODE REVIEW

File: jcliff.spec [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+%{_datadir}/jcliff/COPYING
+%{_datadir}/jcliff/README.markdown
+%{_datadir}/jcliff/cookcc-0.3.3.jar
+%{_datadir}/jcliff/jboss-dmr-1.1.1.Final.jar
+%{_datadir}/jcliff/jcliff
+%{_datadir}/jcliff/jcliff-2.12.6
+%{_datadir}/jcliff/jcliff-2.12.6.jar
+%{_datadir}/jcliff/jcliff.bat
+%{_datadir}/jcliff/junit-4.4.jar
+%{_datadir}/jcliff/rules/datasource
```
Comment (rpelisse) [inline jcliff.spec:39]: @hcherukuri do we need to list all the rules? Can't we simply add jcliff/rules/ ?

File: jcliff.spec [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+%files
+%defattr(0644,root,root,0755)
+%{_bindir}/jcliff/jcliff
+%{_datadir}/jcliff/COPYING
+%{_datadir}/jcliff/README.markdown
+%{_datadir}/jcliff/cookcc-0.3.3.jar
+%{_datadir}/jcliff/jboss-dmr-1.1.1.Final.jar
+%{_datadir}/jcliff/jcliff
+%{_datadir}/jcliff/jcliff-2.12.6
```
Comment (rpelisse) [inline jcliff.spec:35]: @hcherukuri Can we generate this part ? Or move all the jars into a deps/ folder and simply include deps/ ? I don't like the idea of editing this SPEC file every time a dependency changes.

**Review by rpelisse — CHANGES_REQUESTED** [review]
@hcherukuri see comments in file :)

File: jcliff.spec (hunk 2 above)
Comment (hcherukuri) [reply jcliff.spec:35]: If we add all the jars in the directory, I checked if we add just the directory then it should work.

File: jcliff.spec (hunk 1 above)
Comment (hcherukuri) [reply jcliff.spec:39]: As mentioned above, adding %{_datadir}/jcliff/rules/ should work.

**Review by hcherukuri — COMMENTED** [review]
Build rpm with the updated spec file. Removed the list of files and replaced with folders.

**Review by rpelisse — APPROVED** [review]

**Pull request merged**